readme.md

```diff
@@ -43,59 +43,71 @@ If you use this software in your work, please cite our JOSS article with the fol
 
 ### Requirements
 
-  * Python 3.6+ with pip.
+- Python 3.8, 3.9, or 3.10 with pip.
 
 We strongly recommend using the Anaconda Python distribution and creating a new conda environment for OpenOA. You can download Anaconda through [their website.](https://www.anaconda.com/products/individual)
 
 After installing Anaconda, create and activate a new conda environment with the name "openoa-env":
 
-```
-conda create --name openoa-env python=3.8
+```bash
+conda create --name openoa-env python=3.10
 conda activate openoa-env
 ```
 
 ### Installation
 
 Clone the repository and install the library and its dependencies using pip:
 
-```
+```bash
 git clone https://github.com/NREL/OpenOA.git
-pip install ./OpenOA
+cd OpenOA
+pip install .
 ```
 
-You should now be able to import openoa from the Python interpreter:
+If you wish to work with the examples, be sure to install the extra dependencies associated with
+them.
 
+```bash
+pip install ".[examples]"
 ```
+
+You should now be able to import openoa from the Python interpreter:
+
+```bash
 python
 >>> import openoa
+>>> openoa.__version__
 ```
 
-#### Common Installation Issues:
+#### Common Installation Issues
 
 - In Windows you may get an error regarding geos_c.dll. To fix this install Shapely using:
 
-```
+```bash
 conda install Shapely
 ```
 
 - In Windows, an ImportError regarding win32api can also occur. This can be resolved by fixing the version of pywin32 as follows:
 
-```
+```bash
 pip install --upgrade pywin32==255
 ```
 
 ### Development
```

Comment:
Good idea, I linked to the place where it will be once these updates are live and in v3 so we don't have to change it again, so the documentation site link will break if you try to go to it